torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -0,0 +1,187 @@
+#include <torch/csrc/jit/passes/onnx/constant_fold.h>
+#include <c10/util/Exception.h>
+
+#include <c10/util/Optional.h>
+#include <algorithm> 
+
+#if defined(_MSC_VER)
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
+#endif
+
+namespace torch {
+namespace jit {
+
+namespace onnx {
+using namespace ::c10::onnx;
+}
+
+using ParamMap = std::map<std::string, at::Tensor>;
+using ValueToParamPairMap = std::map<Value*, std::pair<std::string, at::Tensor>>;
+
+static ValueToParamPairMap buildValueToParamsMap(Block* b, const ParamMap& paramsDict) {
```

Comment:
We can put everything into anonymous namespace except `ConstantFoldONNX`. So they don't need to be static i think.

Reply:
I think both are viable and have the same effect. But your suggestion is cleaner. Updated with anonymous namespace.